pkg/arvo/sys/zuse.hoon

```diff
@@ -5453,6 +5453,35 @@
     |=  cis/coin
     ?.  ?=({$$ case} cis)  ~
     `(unit beam)`[~ [who dex `case`p.cis] (flop t.t.t.pax)]
+  ::
+  ++  json-rn                                           ::  json to rn parser
+    ;~  plug
```

Comment:
We haven't done this consistently, but it's nice to explicitly cast a parser so it's obvious what it produces. A parser is a wet gate from `+nail` to `+edge`, and `+like` is a mold-builder for a specific type of `+edge`. If I'm reading this correctly, it should be:

```
^-  $-(nail (like dn))
```

Reply:
Fixed!

Reply:
Doesn't `$-` break parser composition? If you can still plug the resulting parser that's great, otherwise iirc `++knee` is the conventional way to add this sort of annotation

Reply:
@ohAitch, you're right. I've used this convention in some of my own parsers, but only at the top level, where they were never being composed.

@pkova, sorry about this -- you can just delete the line, or replace it with

```
%+  knee  *rn  |.
```

Reply:
And done!